ignite/contrib/engines/common.py

```diff
@@ -257,10 +262,14 @@ def setup_any_logging(logger, logger_module, trainer, optimizers, evaluators, lo
     )
 
 
-def _setup_logging(logger, trainer, optimizers, evaluators, log_every_iters):
+def _setup_logging(
+    logger: BaseLogger,
+    trainer: Engine,
+    optimizers: Optional[Union[Optimizer, Dict[str, Optimizer]]],
```

Comment:
this line and 2 below ones are not Optional

Reply:
Done